agency/README.md

```diff
@@ -373,6 +373,7 @@ The Trips endpoint serves two purposes:
 |-------------------------------|--------------------------------|------------------------| ----------------- |
 | trip_id                       | UUID                           | Required               | UUID for the trip this payload pertains to |
 | journey_id                    | UUID                           | Optional               | A unique ID for associating collections of trips |
+| trip_type                     | Enum                           | Optional               | The type of the trip |
```

Comment:
Here a trip_type is added in order to precise what kind of segment a vehicle is on (picking up the delivery, delivering, roaming, ...).